infra/src/main/kotlin/fi/fta/geoviite/infra/ratko/RatkoLocalService.kt

```diff
@@ -55,4 +57,13 @@ class RatkoLocalService @Autowired constructor(
     fun getOperatingPoints(bbox: BoundingBox): List<RatkoOperatingPoint> {
         return ratkoOperatingPointDao.getOperatingPoints(bbox)
     }
+
+    fun searchOperatingPoints(searchTerm: FreeText, resultLimit: Int = 10): List<RatkoOperatingPoint> {
+        logger.serviceCall(
+            "searchOperatingPoints",
+            "searchTerm" to searchTerm,
+            "resultLimit" to resultLimit,
+        )
+        return ratkoOperatingPointDao.searchOperatingPoints(searchTerm, resultLimit)
+    }
```

Comment:
Mietin olisiko täällä hyvä vai huono jos käyttäisi identtistä patternia nykyisten hakujen kanssa... Eli käytännössä että täällä servicessä olisi vaan kaikki liikennepaikat palauttava `.list()`-funktio, jota sitten filtteröitäisiin myöhemmin erillisellä filtterifunktiolla. Toisaalta täällä kyllä tunnuttaisiin tekevän tuo hakeminen kantatasolla, joka taas kuulostaisi siltä että _se_ saattaisi olla nykyistä searchia järkevämpi patterni ettei joka haulla tarvitse ladata koko maailmaa. Enivei nämä voisi olla kiva yhtenäistää suuntaan tai toiseen. Joskin mikäli päädytään menemään sillä ratkaisulla että yhdistetään kannassa tehtäväksi hauiksi, niin se on sen verran iso jumppa että ei ehkä kannata tehdä suoraan tämän tiketin alla jollet halua.